src/scripts/SearchButtonField.js

```diff
@@ -106,8 +106,11 @@ export default class SearchButtonField extends React.Component {
   }
 
   onKeyDown(event) {
-    if (event.keyCode === 13 && this.props.onEnter) {
-      this.props.onEnter(event);
+    const { closeOnEscape, onEnter } = this.props;
+    if (event.keyCode === 13 && typeof onEnter === 'function') {
```

Comment:
no need to check if it is function, there is proptypes for that, just check it exists like it was before